packages/kbn-optimizer/limits.yml

```diff
@@ -111,3 +111,4 @@ pageLoadAssetSize:
   cases: 162385
   screenshotMode: 17856
   visTypePie: 35583
+  expressionRevealImage: 62826
```

Comment:
Based on the description of the PR, I would have expected this page load size to be removed from the canvas plugin but it doesn't appear to be the case. The canvas page load bundle is already over 1M, which is loaded on ALL requests to Kibana. Further increasing this size is concerning. 

Ref: https://github.com/elastic/kibana/issues/95857

Reply:
I'm sorry for not meeting your expectations. Right now a lot of code, related to renderers and functions is reused in multiple renderers. And until the time even one renderer/element/ui_view/function is using some function from lib or helper I can't remove it. As it is the first element, which was extracted, the impact of such operation is not so big, but if all/most of renderers will be moved, the size will decrease.

Reply:
And what about total duplication of code, I'm proposing to create separate plugin in OSS are with libs/types/helpers for sharing between expression_* plugins.

Reply:
We chatted over Zoom - I am going to sync with @clintandrewhall and won't block this PR.